Gemfile.lock

```diff
@@ -105,7 +105,6 @@ PATH
 
 GEM
   remote: https://rubygems.org/
-  remote: https://enterprise.contribsys.com/
```

Comment:
Make sure the sidekiq enterprise gems aren't removed

Reply:
Not sure why that happened because I have the enterprise env var set